chapters/annotations.tex

```diff
@@ -109,10 +109,11 @@ \subsection{Annotations for Figures}\label{annotations-for-figures}
 The \lstinline!identifier! in \lstinline!Figure! and \lstinline!Plot! is an optional \lstinline!String! identifier, and is intended to identify the
 \lstinline!Figure! and \lstinline!Plot! for programmatic access.  The \lstinline!figures! annotation is inherited in the sense that each class has
 a collection of figures comprised by the contents of the \lstinline!figures! annotation in the class itself, as well as the \lstinline!figures! annotations
-from any base classes.  A \lstinline!Figure! must be uniquely identified by its \lstinline!identifier! and a class having it in its collection.  This
+from any base classes. A \lstinline!Figure! must be uniquely identified by its \lstinline!identifier! and a class having it in its collection.  This
```

Comment:
If touch formatting here, we should simply turn the paragraph into sentence-based line breaking.

Reply:
Looks good now.  Resolving.